cmake/cc.cmake

```diff
@@ -36,9 +36,9 @@ if (${CMAKE_C_COMPILER_ID} STREQUAL "GNU" OR ${CMAKE_C_COMPILER_ID} STREQUAL "LS
 
     if (LOONGARCH64)
       if (BINARY64)
-        set(CCOMMON_OPT "${CCOMMON_OPT} -mabi=lp64")
+        set(CCOMMON_OPT "${CCOMMON_OPT} -mabi=lp64d")
       else ()
-        set(CCOMMON_OPT "${CCOMMON_OPT} -mabi=lp32")
+        set(CCOMMON_OPT "${CCOMMON_OPT} -mabi=ilp32d")
```

Comment:
It's better to add a compiler check here? use new abi from a certain version.

Reply:
Done, waiting for the CI results.